xml/schema/decoder-4-15-2.xsd

```diff
@@ -918,7 +918,7 @@
             <xs:attribute name="writeOnly" type="yesNoType" default="no" />
             <xs:attribute name="default" type="xs:string"/>
             <xs:attribute name="comment" type="xs:string"/>
-            <xs:attribute name="item" type="xs:string"/><!-- TODO should be use="required" !!! -->
+            <xs:attribute name="item" type="xs:string" use="required"/><!-- required since 4.23.3 -->
```

Comment:
This change would affect the ability of (some?) old XML files to "validate".  Shouldn't this change be made in a newly-versioned XSD file, such as 'decoder-4-23-3.xsd', instead of 'decoder-4-15-2.xsd'?

Reply:
As discussed, any decoder that uses a qualifier needs the items attribs to be present. I now have all decoder files passing item=required in their (existing) xsd schema.